lib/pwned_password_downloader.rb

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env ruby
+require 'fileutils'
+require 'net/http/persistent'
+require 'ruby-progressbar'
+
+class PwnedPasswordDownload
```

Comment:
what do you think of adding specs for this? a little webmock + tmpdir like this?

https://github.com/18F/identity-idp/blob/a295f125ed558b94200c806855f2a21b80fbc896/spec/bin/oncall/download-piv-certs_spec.rb#L63-L69

I can also help add spec if needed

Reply:
Yep I hadn't got around to the specs yet but was starting to think about them, thanks for the pointer on the PIV downloader specs.

Reply:
Specs are still in-progress, but got the basics started in 371df2768d8fb51241496593e04e197e435fc186.